roadmap/c-unwind-abi.md

```diff
@@ -36,3 +39,14 @@ you will have to keep up. Effectively, you're on a nightly release,
 even though you're using only stable syntax. (The same is true for
 many aspects of unsafe code.)
 
+[LLVM-UB]: ../spec-terminology.md#LLVM-undefined-behavior-or-LLVM-UB
+
+## Panic = abort
+
+In order to safely call functions that may unwind, a Rust function must have
+a landing pad. Typically, `panic = abort` prevents landing pads from being
+generated, which would make the behavior of `"C unwind"`
+[LLVM-undefined][LLVM-UB]. To make this behavior well defined, we would like to
+require landing-pad generation for any function calling a `"C unwind"`
+function, even when compiling with `panic = abort`. These landing pads would of
+course `abort` the application rather than propagate the unwind.
```

Comment:
This is discussing a future in which it's safe to call functions that _can_ unwind using `extern "C unwind"`. (Even if the function is in Rust, it could, for instance, be in a shared library compiled with `panic = unwind`.)

Reply:
If calling an `extern "C unwind"` function cannot unwind, then this feature adds no value. For this feature to add any value to the language, such functions must be allowed to unwind.